pkg/app/piped/platformprovider/kubernetes/testdata/testkustomize-with-helm/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+# Test if kustomize can use the specified version of Helm.
+# This would fail if it uses the default version (3.8.2).
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+helmCharts:
+  - name: datadog
+    releaseName: datadog-agent
+    repo: https://helm.datadoghq.com
+    version: 3.102.0
```

Comment:
Perhaps we could come up with a minimal testcase and put it on local, would like to leave the decision to the maintainers.